src/About/About.tsx

```diff
@@ -1,5 +1,5 @@
 import './About.scss'
-import React, { useState, useEffect } from "react";
+import React from "react";
```

Comment:
Thank you for going through and removing those linter errors

Reply:
<3